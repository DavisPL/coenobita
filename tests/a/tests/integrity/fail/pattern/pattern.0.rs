#![allow(unused_variables)]
#![allow(unused_assignments)]

fn main() {
    #[cnbt::tag({*}{*} ({bin}{bin}, {b}{b}))]
    let (x, y) = (5, 6);
}