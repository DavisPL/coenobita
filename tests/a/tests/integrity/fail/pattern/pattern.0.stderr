error: expected {*}{*} ({bin}{bin},{b}{b}), found {bin}{bin} ({bin}{bin},{bin}{bin})
 --> tests/integrity/fail/pattern/pattern.0.rs:6:18
  |
6 |     let (x, y) = (5, 6);
  |                  ^^^^^^
